tests/components/awair/test_config_flow.py

```diff
@@ -144,27 +144,32 @@ async def test_reauth(hass: HomeAssistant, user, cloud_devices) -> None:
     with patch("python_awair.AwairClient.query", side_effect=AuthError()):
         result = await hass.config_entries.flow.async_configure(
             result["flow_id"],
-            user_input=CLOUD_CONFIG,
+            user_input={CONF_ACCESS_TOKEN: "bad"},
         )
 
-        assert result["type"] is FlowResultType.FORM
-        assert result["step_id"] == "reauth_confirm"
-        assert result["errors"] == {CONF_ACCESS_TOKEN: "invalid_access_token"}
+    assert result["type"] is FlowResultType.FORM
+    assert result["step_id"] == "reauth_confirm"
+    assert result["errors"] == {CONF_ACCESS_TOKEN: "invalid_access_token"}
 
     with (
         patch(
             "python_awair.AwairClient.query",
             side_effect=[user, cloud_devices],
         ),
-        patch("homeassistant.components.awair.async_setup_entry", return_value=True),
```

Comment:
This patch implied that the reload was expected.
Now it actually does reload and we check that it did...